src/main/java/io/epirus/console/project/utils/ProjectUtils.java

```diff
@@ -48,4 +62,97 @@ static void deleteFile(File fileToDelete) {
+                            .collect(Collectors.toList())
+                            .get(0));
+        } catch (IOException e) {
+            Console.exitError(
+                    "Could not load wallet file. Make sure you are in the right directory.");
+            return Optional.empty();
+        }
+    }
+
+    public static Optional<Path> loadProjectPasswordWalletFile(Path path) {
+        String pathToProjectResources =
+                String.join(File.separator, path.toString(), "src", "test", "resources", "wallet");
+        try {
+            return Optional.of(
+                    Files.list(Paths.get(pathToProjectResources))
+                            .filter(f -> f.getFileName().toString().endsWith("-Password"))
+                            .collect(Collectors.toList())
+                            .get(0));
+        } catch (IOException e) {
+            Console.exitError(
+                    "Could not load wallet password file. Make sure you are in the right directory.");
+
+            return Optional.empty();
+        }
+    }
+
+    public static Credentials createCredentials(Path walletPath, Path walletPasswordPath) {
+        String walletPassword = null;
+        try {
+            walletPassword = new String(Files.readAllBytes(walletPasswordPath));
+        } catch (IOException e) {
+            Console.exitError("Could not read password file.");
+        }
+        try {
+            return WalletUtils.loadCredentials(walletPassword, walletPath.toFile());
+        } catch (IOException e) {
+            Console.exitError("Could not create credentials.");
+        } catch (CipherException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    public static void uploadSolidityMetadata(Network network, Path workingDirectory) {
+        File pathToMetadata =
+                new File(
+                        String.join(
+                                File.separator,
+                                workingDirectory.toString(),
+                                "build",
+                                "resources",
+                                "main",
+                                "solidity"));
+        if (pathToMetadata.exists()) {
+            Arrays.stream(pathToMetadata.listFiles())
+                    .filter(file -> file.getName().contains("_meta"))
+                    .forEach(
+                            file -> {
+                                try {
+                                    uploadFile(file, network);
+                                } catch (IOException e) {
+                                    e.printStackTrace();
+                                }
+                            });
+        } else {
+            Console.exitError(
+                    "Could not find the metadata files in :" + pathToMetadata.getAbsolutePath());
+        }
+    }
+
+    private static void uploadFile(File file, Network network) throws IOException {
```

Comment:
I think this is fine for now, but we should probably have a conversation about how we deal with this going forward.